crates/driver/src/print_build.rs

```diff
@@ -668,11 +723,11 @@ where
 
     fn append_key_values(
         extra_kv_pairs: &mut HashMap<String, Vec<String>>,
-        key: &String,
+        key: String,
         values: &Vec<String>,
     ) {
         extra_kv_pairs
-            .entry(key.clone())
```

Comment:
if we unconditionally clone, just accept the `String` so the caller can possibly avoid the clone.